tensorflow/inference/docker/2.8/py3/cu112/Dockerfile.gpu

```diff
@@ -74,6 +74,8 @@ RUN apt-get update \
     tk-dev \
     libgdbm-dev \
     libc6-dev \
+    # remove libsasl2-2 after Nvidia docker upstream fix CVE-2022-24407
+    libsasl2-2 \
```

Comment:
Can we use the `-only-upgrade` solution as done in PR1685. It works for e3 containers as well?